views/networking/load_balancer/overview.erb

```diff
@@ -13,6 +13,23 @@
     ["Stack", @lb.stack.gsub("ip", "IP")],
     ["Load Balancer Port", @lb.src_port],
     ["Application Port", @lb.dst_port],
-    ["HTTP Health Check Endpoint", @lb.health_check_endpoint]
+    ["HTTP Health Check Endpoint", @lb.health_check_endpoint],
+    ["Health Check Protocol", @lb.health_check_protocol.upcase],
```

Comment:
This is handled better by #3889, which has been waiting for your review for 3 weeks. Can you review/approve that, and rebase this on top of that?